webtau-config/src/main/java/org/testingisdocumenting/webtau/cfg/WebTauConfig.java

```diff
@@ -48,6 +48,11 @@
     private final ConfigValue config = declare("config", "config file path", () -> CONFIG_FILE_NAME_DEFAULT);
     private final ConfigValue env = declare("env", "environment id", () -> "local");
     private final ConfigValue url = declare("url", "base url for application under test", NO_DEFAULT);
+
+    private final ConfigValue httpProxy = declare("httpProxy", "http proxy host:port", NO_DEFAULT);
+    private final ConfigValue httpsProxy = declare("httpsProxy", "https proxy host:port", NO_DEFAULT);
+    private final ConfigValue noProxy = declare("noProxy", "list of hosts to exclude from proxy host1|*.host2|host3.*", NO_DEFAULT);
```

Comment:
How about `noProxyHosts`?  If I saw `noProxy` without the description, I'd assume it was a boolean toggle, it's not clear this is a list of hosts from the name.

Reply:
thanks, makes sense